### ISSUE TITLE
Peer Review Final Report - atp44

### ISSUE THREAD

**Issue by anthon11786** [GaryLeeONE/BaseballSTW#19]
This project aimed to predict whether a professional baseball player would, swing, whiff, or take a given pitch within a game. The team had an impressive dataset with over 700k entries in each row representing one pitch during a game. The data represented only one season, but given how many pitches in a season this is still plenty. I really loved the visualizations the team created especially during the exploratory data analysis section. Especially, since I’m familiar with baseball, seeing the heat maps of strike zones is common even during a televised game, so I really liked the hexagon heat maps showing accuracy. I thought it was very intuitive and telling. Also, the team did a very good job of explaining a model we didn’t cover in class, XGBoost, to someone (me) who wasn’t already familiar with the model. The parameters and scope of the algorithm were included and were very helpful for me to digest the new model. 

I thought some areas of improvement could be when discussing if the model has the potential to be a ‘Weapon of Math Destruction’ and the fairness of the model. There did not seem to be any discussion on this so in the future this topic could be added to the report. Additionally, for model selection, maybe try another model as well because there wasn’t much experimentation for different types of models or their loss functions or regularizers, etc.  Overall, I thought it was a very cool report and as a former baseball player, enjoyed it!
